docs/photoniq/prerendering/best-practices-prerendering.md

```diff
@@ -5,26 +5,22 @@ title: Prerendering Best Practices
 
 Prerendering is an optimization technique that generates static HTML pages from dynamic content to improve website performance. The following best practices help you get the most out of Prerendering.
 
-## Selective Prerendering
-
-To maximize the benefits of prerendering, target specific areas of your website that will gain the most from this optimization technique. Typically, these are high-traffic pages or sections featuring dynamic content that seldom changes. By focusing on these areas, you ensure efficient use of resources and faster load times for your users.
-
 ## Domain Configuration
 
 Be very careful when configuring the domains for prerendering. Incorrect settings can result in caching problems and the delivery of outdated content. Make sure the specified domains are accurate and correspond to the sections of your site that you intend to prerender.
 
 ## Use the API for Caching and Recaching
 
-Take advantage of the prerendering service's API to manually cache or recache pages. This ensures that crawlers always access up-to-date content, and allows you greater control over your caching strategy.
+Take advantage of the prerendering service's API to programatically cache or recache pages. This ensures that crawlers always access up-to-date content, and allows you greater control over your caching strategy. You can also manually purge the cache in the web console.
 
-## Configure Ignored Parameters
-
-Though a default set of tracking parameters to be ignored is provided, you may need to customize this list based on your unique needs. For instance, if your website uses specific URL parameters for user sessions or tracking, add these to the list of parameters that should be ignored during prerendering.
+For more information about managing the Preredering cache, refer to [Manage Cache](manage-cache).
 
 ## Remove or Update Dead Links
 
 Dead links negatively affect both your SEO rankings and the cost of prerendering. Regularly scan your website for dead links and either remove them or update them to point to active pages. This improves user experience and ensures you're not incurring extra charges for prerendering pages that don't contribute to your site's value.
 
+If you have a system that redirects users to a page that provides a 404 experience if they stumble on something that isn't covered, then you might want to add a 404 identifier to Prerendering. For more information, refer to [Manage 404 Identifiers](manage-404-identifiers).
+
 ## Implement User-Agent String Detection on Your Origin Server
```

Comment:
I think we should remove this section or rewrite it to mention that PhotonIQ Prerendering can also be configured to render pages as a mobile device. So here, it is important that the website supports mobile devices.

Reply:
Well, which is it? Remove it or rewrite it? I'm kind of fuzzy on user-agent strings, are those what tells the page if the bot/user is on desktop or mobile?

Reply:
Yeah, I think we should remove it.

The only use case for the customer to check the user agent is to identify if the request is coming from a mobile device so they can redirect to a different URL, which is entirely different.

Akamai is verifying the user-agent header to redirect bot requests to PhotonIQ Prerender, so the customer does not need to worry about it.